docs/QSG/docker.md

```diff
@@ -1,172 +1,87 @@
-specifying the "hugepages=N" parameter, where 'N' = the number of huge pages requested.  It is also
-possible to allocate them at run time, thanks to the `sysctl` command:
-
-```bash
-$ sysctl vm.nr_hugepages=8192
-```
-
-It is also possible to use the `sysctl` command to allocate huge pages at boot time with the
-following command:
-
-```bash
-$ cat <<< "vm.nr_hugepages = 8192" > /etc/sysctl.d/50-hugepages.conf
-$ sysctl -p
-```
+If hugepages have not been enabled, we will need to do so now to avoid memory fragmentation. To configure 2Mb hugepages use the following command:
+- ubuntu ```$ sysctl vm.nr_hugepages=4096```
 
 ## Building Docker Images
-
-### Base DAOS Image
-
-The first image to create is the `daos-base` image which is not intetended to be used as it, but as
-a base image for building the other three daos images.  This first image could be built directly
-from GitHub with the following command:
-
-```bash
-$ docker build --tag daos-base:rocky8.4 \
-	https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8
-```
-
-This Docker file accept the following arguments:
-
-- `RHEL_BASE_IMAGE`: Base docker image to use (default "rockylinux/rockylinux")
-- `RHEL_BASE_VERSION`: Version of the base docker image to use (default "8.4")
-- `BUST_CACHE`: Manage docker building cache (default undefined).  To invalidate the cache, a random
-	value such as the date of the day shall be given.
-- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "yes")
-- `DAOS_REPOS`: Space separated list of repos needed to install DAOS (default
-	"https://packages.daos.io/v2.0/CentOS8/packages/x86_64/")
-- `DAOS_GPG_KEYS`: Space separated list of GPG keys associated with DAOS repos (default
-	"https://packages.daos.io/RPM-GPG-KEY")
-- `DAOS_REPOS_NOAUTH`: Space separated list of repos to use without GPG authentication
-	(default "")
-
-For example, building a DAOS base image, with authentication disabled, could be done with the
-following command:
-
-```bash
-$ docker build --tag daos-base:rocky8.4 --build-arg DAOS_AUTH=no \
-	https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8
-```
-
-It is also possible to build the `daos-base` image from a local tree with the following command:
-
-```bash
-$ docker build --tag daos-base:rocky8.4 utils/docker/vcluster/daos-base/el8
-```
-
-### DAOS Nodes Images
-
 The three images `daos-server`, `daos-admin` and `daos-client` could be built directly from GitHub
-or from a local tree in the same way as for the `daos-base` image.  Following command could be used
-to build directly the three images from GitHub:
+or from a local tree in the same way as for the `daos-base` image.
 
+Following command could be used to build directly the three images from GitHub:
+
+@@@ as far as can determine the tag doesnt exist	
 ```bash
 $ for image in daos-server daos-admin daos-client ; do \
```

Comment:
If the goal is to simplify the procedure, using docker-compose should be favored